.travis.yml

```diff
@@ -1,24 +1,22 @@
 language: go
 
 go:
-    - "1.11"
-    - master
+  - "1.11.x"
```

Comment:
You should also specify a minor version.

Reply:
Hmm, `1.11.x` is used as a latest patch version(i.e. 1.11.5), I think.  Is it better to specify a version, or is my understanding wrong?

Reply:
You should avoid using latest.
You should do it manually if the minor version goes up.

Reply:
I got it, fixed.